ethics/ethics.md

```diff
@@ -30,13 +30,14 @@ Software is advancing in many aspects, from pure technological capabilities to u
 - [ACM Code of Ethics and Professional Conduct](https://ethics.acm.org/)
 - [Introduction to 
 Software Engineering Ethics](https://www.scu.edu/media/ethics-center/technology-ethics/Students.pdf)
```

Comment:
The lesson and guided practice actually do use these materials. Please change it back.